tests/test_artifactory.py

```diff
@@ -4,41 +4,46 @@
 import audeer
 
 
-BACKEND = audbackend.Artifactory(
-    pytest.HOSTS['artifactory'],
-    pytest.REPOSITORIES['artifactory'],
-)
+@pytest.fixture(scope='function')
+def backend(request):
 
+    backend = audbackend.Artifactory(
+        pytest.HOSTS['artifactory'],
+        f'unittest-{audeer.uid()[:8]}',
+    )
+
+    yield backend
+
+    # TODO: replace with audbackend.delete() when available
```

Comment:
It would be handy to introduce such a function. But it's also somehow dangerous of course :)

Reply:
It depends, you could argue that `audbackend` is for developers anyway.